state-chain/pallets/cf-ingress-egress/src/mock_eth.rs

```diff
@@ -78,6 +78,18 @@ impl OnDeposit<Ethereum> for MockDepositHandler {}
 pub type MockEgressBroadcaster =
 	MockBroadcaster<(MockEthereumApiCall<MockEvmEnvironment>, RuntimeCall)>;
 
+impl LpDepositHandler for MockDepositHandler {
+	type AccountId = u64;
+
+	fn add_deposit(
+		_who: &Self::AccountId,
+		_asset: Asset,
+		_amount: AssetAmount,
+	) -> frame_support::pallet_prelude::DispatchResult {
+		Ok(())
```

Comment:
We can create a better mock than this... Something like:

```suggestion
		MockBalance::try_credit_account(account_id, asset, amount)?;
		Ok(())
```

(And then the tests can check the balance.)